website/docs/cli/workspaces/index.mdx

```diff
@@ -7,72 +7,99 @@ description: >-
-resources with the same configuration.
-
-- Every [initialized working directory](/cli/init) has at least
-  one workspace. (If you haven't created other workspaces, it is a workspace
-  named `default`.)
-- For a given working directory, only one workspace can be _selected_ at a time.
-- Most Terraform commands (including [provisioning](/cli/run)
-  and [state manipulation](/cli/state) commands) only interact
-  with the currently selected workspace.
-- Use [the `terraform workspace select` command](/cli/commands/workspace/select)
-  to change the currently selected workspace.
-- Use the [`terraform workspace list`](/cli/commands/workspace/list),
-  [`terraform workspace new`](/cli/commands/workspace/new), and
-  [`terraform workspace delete`](/cli/commands/workspace/delete) commands
-  to manage the available workspaces in the current working directory.
-
--> **Note:** Terraform Cloud and Terraform CLI both have features called
-"workspaces," but they're slightly different. Terraform Cloud's workspaces
-behave more like completely separate working directories.
-
-## The Purpose of Workspaces
-
-Since most of the resources you can manage with Terraform don't include a unique
-name as part of their configuration, it's common to use the same Terraform
-configuration to provision multiple groups of similar resources.
-
-Terraform relies on [state](/language/state) to associate resources with
-real-world objects, so if you run the same configuration multiple times with
-completely separate state data, Terraform can manage many non-overlapping groups
-of resources. In some cases you'll want to change
-[variable values](/language/values/variables) for these different
-resource collections (like when specifying differences between staging and
-production deployments), and in other cases you might just want many instances
-of a particular infrastructure pattern.
-
-The simplest way to maintain multiple instances of a configuration with
-completely separate state data is to use multiple
-[working directories](/cli/init) (with different
-[backend](/language/settings/backends/configuration) configurations per directory, if you
-aren't using the default `local` backend).
-
-However, this isn't always the most _convenient_ way to handle separate states.
-Terraform installs a separate cache of plugins and modules for each working
-directory, so maintaining multiple directories can waste bandwidth and disk
-space. You must also update your configuration code from version control
-separately for each directory, reinitialize each directory separately when
-changing the configuration, etc.
-
-Workspaces allow you to use the same working copy of your configuration and the
-same plugin and module caches, while still keeping separate states for each
-collection of resources you manage.
+Workspaces in the Terraform CLI refer to separate instances of [state data](/language/state) inside the same working directory.
```

Comment:
I assume by including "in the Terraform CLI" here you were implicitly saying we're not talking about workspaces in Terraform Cloud, but I would recommend being more explicit about that because it's a common source of confusion.

The common way folks talk about it in the community is to assert that Terraform CLI workspaces and Terraform Cloud workspaces are _entirely separate_. That isn't strictly true -- Terraform Cloud workspaces are more like an _extension_ to Terraform CLI workspaces with a number of additional features and better usability -- but since we are intentionally not talking about the Terraform Cloud-unique features here I think it would be helpful to link right up top to some more relevant page to read if you are a Terraform Cloud user, such as [the Cloud Workspaces overview page](https://www.terraform.io/cloud-docs/workspaces).

Reply:
This is a good point --> I can push up a fix for this!